content/applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst

```diff
@@ -1,101 +1,256 @@
+:guilabel:`Time Range` units shown on the |MPS| page. For example, if the :guilabel:`Time Range`
+field is set to :guilabel:`Monthly`, and `12` is entered in the :guilabel:`Number of Columns` field,
+the |MPS| shows one column for the next 12 months, starting with the current month.
+
+If the values of the :guilabel:`Time Range` or :guilabel:`Number of Columns` fields are altered,
+click :guilabel:`Save` again to save the changes.
+
+.. image:: use_mps/mps-setting.png
+   :align: center
+   :alt: The MPS setting in the Manufacturing app's settings.
+
+MPS UI
+======
 
-Go to the :menuselection:`Manufacturing app --> Configuration --> Settings` and
-activate the Master Production Schedule feature before hitting save.
+To open the |MPS|, navigate to :menuselection:`Manufacturing app --> Planning --> Master Production
+Schedule`. The |MPS| view appears as follows:
+
+.. image:: use_mps/mps.png
+   :align: center
+   :alt: The Master Production Schedule in the Manufacturing app.
+
+The grey column on the left side of the screen shows a section for every product added to the |MPS|.
+The categories shown in the rows of each product's section depend on the filters selected in the
+:guilabel:`Search...` bar drop-down menu at the top of the page. The default categories are as
+follows:
+
+- :guilabel:`[Product] by [unit]` :icon:`fa-area-chart`: the forecasted stock quantity at the
+  beginning of each time period. :guilabel:`[Product]` and :icon:`fa-area-chart` are selectable
+  buttons which open the product's page or the forecast report for the product, respectively.
+- :guilabel:`- Forecasted Demand`: the demand forecast, which is entered manually. This represents
+  an estimate of the demand for the product during each time period.
+- :guilabel:`- Indirect Demand Forecast`: while this is a default category, it only appears for
+  products that are components of other products. It represents the demand for the component from
+  existing MOs.
+- :guilabel:`+ Suggested Replenishment`: the quantity of the product that is suggested to be
+  replenished through |MOs| or purchase orders |POs|. To the right of the category title is a
+  :guilabel:`Replenish` button, which is used to manually replenish the product, based on the
+  quantity suggested to be replenished.
+- :guilabel:`= Forecasted Stock`: the quantity of the product forecasted to be in stock at the end
+  of each time period, assuming that suggested replenishment numbers are fulfilled.
+
+Altogether, these default categories form an equation: `Forecasted Demand + Suggested Replenishment
+= Forecasted Stock`. The :guilabel:`- Forecasted Demand` and :guilabel:`+ Suggested Replenishment`
+fields can be edited for any of the time periods to the right of the product column. Doing so
+changes the equation, and updates the value displayed in the :guilabel:`Forecasted Stock` field.
+
+Changing any of the values also makes an :icon:`fa-times` :guilabel:`reset` button appear to the
```

Comment:
I didn't know which value you were talking about at first, so I think it'll be useful to specify here:
```suggestion
In the suggested replenishment line, changing any of the values also makes an :icon:`fa-times` :guilabel:`reset` button appear to the
```